themes/default/content/_index.md

```diff
@@ -40,39 +40,193 @@ overview:
       - /logos/tech/ci-cd/travis-ci.svg
 
 build:
-  title: Build
+  title: Code faster
   description: |
-    Build cloud applications and infrastructure by combining the safety and reliability of [infrastructure as code](/what-is/what-is-infrastructure-as-code/) with the power of familiar programming languages and tools.
+    Pulumi gives you a faster inner dev loop for IaC by allowing you to use your existing IDE with Intellisense and unit tests working out the box. You also can tap into the power of programming languages to model your infrastructure.
   items:
-    - title: Flexible and expressive
-      description: Loops, conditionals, functions, classes, and more.
-
-    - title: Be productive
-      description: Gets things done in seconds rather than hours.
-
     - title: Share and reuse
-      description: Define and consume patterns and practices to reduce boilerplate.
-
-use_cases:
-  - title: All architectures welcome
-    icon: nodes
-    icon_color: purple
-    description: "Choose from over 60 cloud providers, including public, private, and hybrid architectures."
+      description: |
+        You can use Pulumi Packages to build best-practice abstractions available in all supported languages.
 
-  - title: Modernize your approach
-    icon: download-from-cloud
-    icon_color: yellow
-    description: "Adopt serverless, containers, and managed services, and bring them closer to development."
+    - title: Be productive
+      description: |
+        Reduce the time spent switch between by docs in browser and code IDE with Intellisense supported
+        out of the box.
 
-  - title: Adopt Kubernetes
-    icon: guage
-    icon_color: salmon
-    description: "Get up and running with clusters, supporting infrastructure, and application workloads."
+    - title: Flexible and expressive
```

Comment:
Feels like this should be #1. It's the most "simple, obvious, but not obvious" one. The others are more exotic.